code/game/objects/items/storage/belt.dm

```diff
@@ -1122,6 +1122,23 @@
 	for(var/i = 1 to storage_slots - 1)
 		new /obj/item/ammo_magazine/pistol/highpower/black(src)
 
+/obj/item/storage/belt/gun/m39
+	name = "\improper M276 pattern M39 armor rig"
+	desc = "Special issue variant of the M276 designed to holster a M39 submachine gun and two spare magazines. Uncommonly issued to USCM support and specialist personnel, can only be clipped to a set of armor."
+	icon_state = "m39_armor"
+	item_state = "s_marinebelt"
+	flags_equip_slot = SLOT_SUIT_STORE
```

Comment:
Don't do this here, add this to the base marine armor in marine_armor.dm in "allowed" line 100